src/lib/TranscriptEditor/TimedTextEditor/index.js

```diff
@@ -222,6 +226,104 @@ class TimedTextEditor extends React.Component {
     return currentWord;
   }
 
+  myKeyBindingFn = ( e) => {
```

Comment:
Is this just going to be for the enter key? Needs better naming if so I think.

Reply:
for now yes, but `keyBindingFn` could be used for other key combinations as well, so wouldn't make it too specific for now

Reply:
https://draftjs.org/docs/advanced-topics-key-bindings